data/sql/updates/pending_db_world/rev_1629326440508086998.sql

```diff
@@ -0,0 +1,22 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1629326440508086998');
+DELETE FROM `skinning_loot_template` WHERE `Entry` IN (684, 768, 1713, 10237) AND (`Item` IN (7428));
+
+-- Adjust skinning drop rates for Shadowmaw Panther
+UPDATE `skinning_loot_template` SET `Chance` = 73.4 WHERE `Entry` = 684 AND `Item` = 4234;
+UPDATE `skinning_loot_template` SET `Chance` = 3.3 WHERE `Entry` = 684 AND `Item` = 4235;
+UPDATE `skinning_loot_template` SET `Chance` = 23.3 WHERE `Entry` = 684 AND `Item` = 4304;
+
+-- Adjust skinning drop rates for Shadow Panther
+UPDATE `skinning_loot_template` SET `Chance` = 72.7 WHERE `Entry` = 768 AND `Item` = 4234;
+UPDATE `skinning_loot_template` SET `Chance` = 4.2 WHERE `Entry` = 768 AND `Item` = 4235;
+UPDATE `skinning_loot_template` SET `Chance` = 23.1 WHERE `Entry` = 768 AND `Item` = 4304;
+
+-- Adjust skinning drop rates for Elder Shadowmaw Panther
+UPDATE `skinning_loot_template` SET `Chance` = 72.8 WHERE `Entry` = 1713 AND `Item` = 4234;
+UPDATE `skinning_loot_template` SET `Chance` = 3.3 WHERE `Entry` = 1713 AND `Item` = 4235;
+UPDATE `skinning_loot_template` SET `Chance` = 23.9 WHERE `Entry` = 1713 AND `Item` = 4304;
+
+-- Adjust skinning drop rates for Yor <UNUSED>
+UPDATE `skinning_loot_template` SET `Chance` = 72 WHERE `Entry` = 10237 AND `Item` = 4234;
+UPDATE `skinning_loot_template` SET `Chance` = 4.7 WHERE `Entry` = 10237 AND `Item` = 4235;
+UPDATE `skinning_loot_template` SET `Chance` = 23.3 WHERE `Entry` = 10237 AND `Item` = 4304;
```

Comment:
You need skin drops for an unused npc?

Reply:
how does this matter, im fixing wrong data inside the database that is supposed to be "blizzlike" which its going to be if all entires get fixed. theres a reason why a skinning loot template exists for an unused npc.

Reply:
That's my error, not @iraizo's. I'd actually be okay deleting the skinning drops from this NPC, as Yor was apparently a centaur of some kind, and as a humanoid (type 7) they shouldn't be skinnable in any case.

As for being unused, bear in mind AC is used widely by a lot of people with a lot of mods. This NPC might not be used by a vanilla server, but they could quite easily be repurposed by someone for who knows what kind of content sometime down the road. And if they ever do, they'll have correct data.

Reply:
yeah thats why i also wanted to correct it, so whats the option now, leave it like this or delete the skinning entry @Azcobu

Reply:
Going on the basis that centaurs shouldn't be skinnable, and no other NPCs are using that table, I think it would best to set Yor's `skinloot` to 0 and then clear the 3 `Entry = 10237` entries from skinning_loot_template. Thanks.